### PR TITLE
Rad Laser Fix

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Fixes up the Rad Laser

- Rad laser no longer makes the user do an attack animation when it's used on someone
- Rad laser can be stored wherever a health analyzer can be stored
 - RIP Metaing it being a rad laser based on how it behaves or where you can put it.

Related
- Fixes up proper restrictions for topic.
- Adds in logging for the rad laser.

### CODE REVIEW

File: code/game/objects/items/devices/traitordevices.dm [hunk 1]
```diff
@@ -131,7 +130,7 @@ effective or pretty fucking useless.
 	popup.open()
 
 /obj/item/device/rad_laser/Topic(href, href_list)
-	if(!in_range(src, usr) || issilicon(usr) || !usr.canmove || usr.restrained())
+	if(!usr.canmove || usr.stat || usr.restrained() || !in_range(loc, usr))
```
Comment (Markolie) [inline code/game/objects/items/devices/traitordevices.dm]: You can just call if(..()). Higher level Topic checks take care of everything.

Comment (Fox-McCloud) [inline code/game/objects/items/devices/traitordevices.dm]: There's not a Topic for device, item, object, atom, or atom/movable--are you sure that would work?

Comment (Markolie) [inline code/game/objects/items/devices/traitordevices.dm]: Yup: https://github.com/ParadiseSS13/Paradise/blob/fe2a3ef960297c6f9a15f4114c2c60634372ad2e/code/game/objects/objs.dm#L24. That uses the Topic checks handled in NanoUI module files.

Comment (Fox-McCloud) [inline code/game/objects/items/devices/traitordevices.dm]: strange; wonder why I couldn't find that.

**Comment by Fox-McCloud** [comment]
There ya go.

**Pull request merged**